src/Message/ApplePayCompleteAuthorizeRequest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace Omnipay\Vindicia\Message;
+
+/**
+ * After the Apple Pay payment sheet is fully loaded on the frontend. The user can authorize a payment 
+ * using Touch or Face ID –– this will grant access to the ApplePayPaymentToken. You can then authorize a payment
+ * using this request with the ApplePayPaymentToken as a parameter along with other information needed for
+ * an authorization (see AuthorizeRequest). No money will be transferred during authorization.
+ * After authorizing a transaction, call capture to complete the transaction and transfer the money.
+ *
+ * You may use other fields from the ApplePayPayment object to fill out billing info.
+ * This request only requires the 'token' field from the ApplePayPaymentToken extracted from the ApplePayPayment
+ * object on the front end. Pass the extracted 'token' to the 'applePayToken' parameter of this class.
+ * 
+ * Example:
+ * <code>
+ *   // set up the gateway
+ *   $gateway = \Omnipay\Omnipay::create('Vindicia ApplePay');
+ *   $gateway->setUsername('your_username');
+ *   $gateway->setPassword('y0ur_p4ssw0rd');
+ *   $gateway->setTestMode(false);
+ *
+ *   // authorize the transaction
+ *   $authorizeResponse = $gateway->completeAuthorize(array(
+ *  *    // ApplePayPaymentToken extracted from ApplePayPayment object.
+ *       // You retrieve the ApplePayPayment object when a user authorizes a payment
+ *       // using the ApplePay payment sheet.
+ *       'applePayToken' => array(
```

Comment:
just doing `'applePayToken' => $apple_pay_token` _might_ be more clear. i'm worried about ppl thinking that they have to parse token and stick it in this array